src/eel_driver.py

```diff
@@ -5,12 +5,41 @@
 import logging
 from logging.handlers import RotatingFileHandler
 from pathlib import Path
+import json
+import requests
+import os
+import psutil
+import base64
 
 import eel
 
 from champselect import websockets
 from utils import ddragon, runes, sum_spells, summoner_info, champ_identifier
 
+@eel.expose
+def get_ranked_names():
+    # Find the LeagueClient.exe process and extract the required information
+    for process in psutil.process_iter():
+        if process.name() == 'LeagueClient.exe':
+            port = process.cmdline()[2].split('=')[1]
+            auth_key = process.cmdline()[1].split('=')[1]
+            break
+
+    # Create URL and headers for the request
+    url = f'https://127.0.0.1:{port}/chat/v5/participants/champ-select'
+    headers = {
+        'Authorization': f"Basic {base64.b64encode(f'riot:{auth_key}'.encode()).decode()}",
+        'Accept': 'application/json'
+    }
+
+    # Get the response from the server
+    response = requests.get(url=url, headers=headers, verify=r"{}\riotgames.pem".format(os.getcwd()))
+
+    # Parse the JSON response and extract the names
+    response_data = json.loads(''.join(chunk.decode() for chunk in response))
```

Comment:
This conversion ends up not being necessary, you can do response.json once you switch to using willump